openmetadata-ui/src/main/resources/ui/src/components/DataQuality/IncidentManager/TestCaseResultTab/TestCaseResultTab.test.tsx

```diff
@@ -21,54 +21,61 @@ import React from 'react';
 import { TestCase } from '../../../../generated/tests/testCase';
 import { checkPermission } from '../../../../utils/PermissionsUtils';
 import TestCaseResultTab from './TestCaseResultTab.component';
-import { TestCaseResultTabProps } from './TestCaseResultTab.interface';
 
-const mockProps: TestCaseResultTabProps = {
-  testCaseData: {
-    id: '1b748634-d24b-4879-9791-289f2f90fc3c',
-    name: 'table_column_count_equals',
+const mockTestCaseData = {
+  id: '1b748634-d24b-4879-9791-289f2f90fc3c',
+  name: 'table_column_count_equals',
+  fullyQualifiedName:
+    'sample_data.ecommerce_db.shopify.dim_address.table_column_count_equals',
+  testDefinition: {
+    id: '48063740-ac35-4854-9ab3-b1b542c820fe',
+    type: 'testDefinition',
+    name: 'tableColumnCountToEqual',
+    fullyQualifiedName: 'tableColumnCountToEqual',
+    displayName: 'Table Column Count To Equal',
+  },
+  entityLink: '<#E::table::sample_data.ecommerce_db.shopify.dim_address>',
+  entityFQN: 'sample_data.ecommerce_db.shopify.dim_address',
+  testSuite: {
+    id: 'fe44ef1a-1b83-4872-bef6-fbd1885986b8',
+    type: 'testSuite',
+    name: 'sample_data.ecommerce_db.shopify.dim_address.testSuite',
```

Comment:
Can we move this to a mock file?

Reply:
I will take care of this later since i need this PR for collate side!